Gemfile

```diff
@@ -58,7 +58,7 @@ gem "ruby-dbus" # For external auth
 gem "ruby-progressbar",               "~>1.7.0",       :require => false
 gem "ruby_parser",                    "~>3.8",         :require => false
 gem "rufus-scheduler",                "~>3.1.3",       :require => false
-gem "rugged",                         "~>0.23.0",      :require => false
+gem "rugged",                         "=0.25.0b10",    :require => false
```

Comment:
@mkanoor should this be now updated to a non-beta version of the gem, such as 0.25.1?  See https://rubygems.org/gems/rugged/versions